### PR TITLE
Added test for log forwarder surviving upgrades

### PR DESCRIPTION
**Pull request by eldios**
Updated test for log forwarder surviving upgrades

## Description
Added a single line of test to verify that log forwarder ConfigMap survives through upgrades.

## Linked tickets and other PRs
https://github.com/gravitational/gravity/issues/1681

### CODE REVIEW

File: docs/testplan.md [hunk 1]
```diff
@@ -548,6 +548,7 @@ Execute `gravity gc`.
 
   Log forwarder settings
   - [ ] Verify that Creating/Deleting/Editing a log forwarder works.
+  - [ ] Verify that after creating a log forwarder resource, it survive upgrades.
```
Comment (r0mant) [inline docs/testplan.md:551]: I don't think this belongs in the UI section to be honest, it's going to be lost here. I would add a separate section for it, or made this test a part of one of the existing upgrade-related scenarios (e.g. one of the things to check after an upgrade).

File: docs/testplan.md [hunk 2]
```diff
@@ -134,6 +134,7 @@ spec:
 
 - [ ] Install a 1-node cluster of previous LTS version.
 - [ ] Upgrade the cluster to the current version.
+  - [ ] Verify that after customizing the log forwarder resource, changes survive the upgrade.
```
Comment (r0mant) [inline docs/testplan.md:137]: In this case, I would also add a step "Set up a log forwarder resource" as a sub-step of the first "install" step.

**Review by r0mant — APPROVED** [review]

File: docs/testplan.md [hunk 3]
```diff
@@ -845,3 +847,4 @@ app.manifest.extensions.logs.disabled: true
 ```
   - [ ] Verify that K8s tab is displayed.
   - [ ] Verify that Nodes screen displays K8s labels.
+i
```
Comment (wadells) [inline docs/testplan.md:850]: This looks spurious.

Comment (eldios) [reply docs/testplan.md:850]: whooops! yeah definitely a leftover

**Pull request merged**